src/clp_ffi_js/ir/StreamReader.hpp

```diff
@@ -1,51 +1,40 @@
 #ifndef CLP_FFI_JS_IR_STREAM_READER_HPP
 #define CLP_FFI_JS_IR_STREAM_READER_HPP
 
-#include <Array.hpp>
+#include <array>
 #include <cstddef>
 #include <memory>
-#include <optional>
-#include <streaming_compression/zstd/Decompressor.hpp>
-#include <vector>
+#include <string_view>
 
-#include <clp/ir/types.hpp>
-#include <clp/TimestampPattern.hpp>
-#include <emscripten/bind.h>
 #include <emscripten/val.h>
 
-#include <clp_ffi_js/ir/LogEventWithLevel.hpp>
-#include <clp_ffi_js/ir/StreamReaderDataContext.hpp>
-
 namespace clp_ffi_js::ir {
-using clp::ir::four_byte_encoded_variable_t;
 
 EMSCRIPTEN_DECLARE_VAL_TYPE(DataArrayTsType);
 EMSCRIPTEN_DECLARE_VAL_TYPE(DecodedResultsTsType);
 EMSCRIPTEN_DECLARE_VAL_TYPE(FilteredLogEventMapTsType);
+EMSCRIPTEN_DECLARE_VAL_TYPE(LogLevelFilterTsType);
+
+constexpr std::array<std::string_view, 4> cIrV1Versions = {"v0.0.1", "v0.0.0", "0.0.1", "0.0.0"};
```

Comment:
Let's also add 0.0.2 which added support for UTC offset shift.
```suggestion
constexpr std::array<std::string_view, 6> cIrV1Versions
        = {"v0.0.2", "v0.0.1", "v0.0.0", "0.0.2", "0.0.1", "0.0.0"};
```